Formula/autorest.rb

```diff
@@ -0,0 +1,21 @@
+class Autorest < Formula
+  desc "Swagger (OpenAPI) Specification code generator"
+  homepage "https://github.com/Azure/autorest"
+  url "https://github.com/Azure/autorest/releases/download/AutoRest-0.16.0/autorest.0.16.0.zip"
+  version "0.16.0.0"
+  sha256 "bc909aaf3db2105a5d709c940c4d120c5b71047539c2068d484ced1be47650c3"
+
+  depends_on "mono"
+
+  def install
+    libexec.install Dir["*"]
+    (bin/"autorest").write <<-EOS.undent
+      #!/bin/bash
+      mono #{libexec}/autorest.exe "$@"
+    EOS
+  end
+
+  test do
+    assert_match "Microsoft (R) AutoRest #{version}", shell_output("#{bin}/autorest | head -1")
```

Comment:
Can’t we write a test that does something more substantial than printing the version?